### PR TITLE
cargo.toml update

### PR DESCRIPTION
**Pull request by zjma**

### CODE REVIEW

File: storage/scratchpad/Cargo.toml [hunk 1]
```diff
@@ -25,16 +25,16 @@ proptest = { workspace = true, optional = true }
...
-jemallocator = { workspace = true }
-
 [dev-dependencies]
 aptos-types = { workspace = true, features = ["fuzzing"] }
 bitvec = { workspace = true }
 once_cell = { workspace = true }
 proptest = { workspace = true }
 rand = { workspace = true }
 
+[target.'cfg(unix)'.dependencies]
```
Comment (msmouse) [inline storage/scratchpad/Cargo.toml:35]: did you mean `[target.'cfg(unix)'.dev-dependencies]`? (in the current form it's just a reorder?)

**Review by msmouse — APPROVED** [review]

**Review by msmouse — DISMISSED** [review]
sorry approved accidentally

File: storage/scratchpad/Cargo.toml (hunk 1 above)
Comment (gregnazario) [reply storage/scratchpad/Cargo.toml:35]: Yeah, let's put dev-dependencies

Comment (zjma) [reply storage/scratchpad/Cargo.toml]: fixed.

**Review by gregnazario — APPROVED** [review]

**Comment by gregnazario** [comment]
I've updated the commit message in the squash to be more descriptive of the change

**Review by msmouse — APPROVED** [review]

**Pull request merged**